pkgs/applications/audio/mopidy/jellyfin.nix

```diff
@@ -23,3 +23,4 @@ python3Packages.buildPythonApplication rec {
     maintainers = [ maintainers.pstn ];
   };
 }
+
```

Comment:
Could you amend your commit without the spurious newline change at the end of the file?

(e.g. remove it and do `git commit -a --amend --no-edit` and `git push --force-with-lease` from your branch)

Reply:
I get an editorconfig error without this newline

Reply:
You had two newlines and it just needed to be one newline. Right now it's zero newlines (see the red circle thing in the diff). Does that make sense?

Reply:
got it, thanks. I seem to have fixed it